### PR TITLE
Update the book store as API Specification

### PR DESCRIPTION
**Pull request by amabdelmongy**
Update the book store as API Specification
Set the base API URL at the central file 'base-api-url.js'

### CODE REVIEW

File: src/store/base-api-url.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function getBaseApiUrl() {
+  return `http://localhost:5000`;
+}
```
Comment (marnika) [inline src/store/base-api-url.js]: I would suggest to not place the url here, but to set the default base url for axios globally.
First, you can create env folder and create **config.local.json** file in it.
In the json file you can set the url.
Then, in App.vue on created() you can get this value and set the  axios.defaults.baseURL.

File: env/config.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mnikolcheva) [inline env/config.json:1]: Because it is a localhost configuration, we can rename the file to config.local.json

Comment (mnikolcheva) [reply env/config.json:1]: But we can do this later as well

Comment (amabdelmongy) [reply env/config.json:1]: Yes in this way we need another file for production. Maybe later do it

Comment (mnikolcheva) [reply env/config.json:1]: Yep

**Review by marnika — APPROVED** [review]

**Pull request merged**